### DISCUSSION TITLE
Read and Modify only JSON Part of GLTF File (NodeIO)

### DISCUSSION THREAD

**Discussion by chrisssssy** [donmccurdy/glTF-Transform#964] [Q&A] [answered]
Hello,
is it possible to load only the JSON Part of a GLTF File without loading the .bin File or any of the textures? I want to load the GLTF, Modify the position, rotation and extras-property of some nodes and save it again, without doing any modifications to the geometry or the textures.
Thanks @donmccurdy for your library, it is working really well so far for me

**Comment by donmccurdy** [comment, marked as answer]
Hi @chrisssssy! The closest thing within the glTF Transform library is `io.readAsJSON('path/to/scene.gltf')`, which gives you the raw JSON, .bin, and textures. But for what you're describing, it may be just as easy not to use glTF Transform — load the `.gltf` with the Node.js `fs` utility, parse it as JSON, modify it, and save back to disk.

glTF Transform cannot itself do anything to the model without loading resources. It won't modify textures or geometry, but the .bin file could come out different due to data layout and padding differences.

**Comment by chrisssssy** [comment]
Thanks for the answer. I will try it like you said.
